lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -11,9 +11,12 @@ defmodule ExUnit.Case do
 
   When used, it accepts the following options:
 
-    * :async - configure this specific test case to able to run in parallel
+    * `:async` - configure this specific test case to able to run in parallel
```

Comment:
Let's move this to the capture logs/io docs then? In the past, you could not do `capture_io :stderr` at all under async: true, I believe this has been fixed though?